activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Don't ignore `DISTICT` selector value when `:all` is passed into `#count`.
```

Comment:
You have a typo on `DISTINCT`

(It's my first "contribution" on Rails project \o/ !)

Reply:
ugh thanks. I typoed it in the code too originally. 🤦‍♀️